### ISSUE TITLE
Error using Docker build method

### ISSUE THREAD

**Issue by hires** [VCVRack/rack-plugin-toolchain#33]
'Unable to find image 'rack-plugin-toolchain:13' locally
docker: Error response from daemon: pull access denied for rack-plugin-toolchain, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.'

Not sure what I'm doing wrong here. I'm running on Ubuntu 22.04 amd64 with Docker 24.0.5.

The only difference from the instructions is that I have a newer MacOS SDK (14.0) since my Mac has the latest XCode and there doesn't seem to be an easy way to downgrade it.

**Comment by cschol** [comment]
Have you started Docker? It looks like the Docker daemon is not running.

**Comment by hires** [comment]
Yes I believe it's running... I use Docker for other development tasks.

**Comment by cschol** [comment]
What does `docker images` show?

**Comment by ode2spot** [comment]
I am also seeing this issue.  Docker is running as seen with `docker ps`

**Comment by cschol** [comment]
Same question: what does `docker images` show? It should show the `rack-plugin-toolchain` image downloaded locally.

**Comment by ode2spot** [comment]
`docker images` shows my docker images for embedded development.  I am running docker in a WSL2 environment and it looks like the older docker 'build' tool is deprecated in favor of 'buildx/BuilkdKit`.  For me the easiest way to get buildx in my WSL2 setup was:

1) in WSL2, `sudo apt remove docker`
2) install DockerDesktop on Windows

Once I did that, I was able to run `make docker-build`.  However, the build failed.  I attached the [docker-build.log](https://github.com/VCVRack/rack-plugin-toolchain/files/14319737/docker-build.log) here.

It looks like I need MacOSX11.1.sdk.tar on my system to use the docker build setup.  Is that correct?

**Comment by cschol** [comment]
Yes. In order to build the entire toolchain incl. macOS support you need an **official** macOS SDK (it does not work with the ones you "find on the internet").
